src/api/java/de/teamlapen/vampirism/api/event/VampirismVillageEvent.java

```diff
@@ -135,4 +153,244 @@ public void setAgressiveVillager(@Nullable IVillageCaptureEntity captureVillager
             this.captureVillager = captureVillager;
         }
     }
+
+    /**
+     * Fired when the Capture process is finished the the Villager should be affected by the faction change
+     * if result is {@link Result#DENY} the Vanilla code is skipped
+     */
+    @HasResult
+    public static class VillagerCaptureFinish extends VampirismVillageEvent {
+
+        private final @Nonnull List<EntityVillager> villager;
+        private final @Nullable IPlayableFaction<?> controllingFaction;
+        private final @Nonnull IPlayableFaction<?> capturingFaction;
+
+        public VillagerCaptureFinish(@Nullable IVampirismVillage village, @Nonnull List<EntityVillager> villagerIn, @Nullable IPlayableFaction<?> controllingFactionIn, @Nonnull IPlayableFaction<?> capturingFactionIn) {
```

Comment:
Probably village should be nonnull here, posting this event for a non existent village does not make much sense